README.md

```diff
@@ -2432,6 +2432,39 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
+ To configure a logger, you need to use the `ClientConfig.properties['hazelcast.logging']` property. If you set it to `'off'`, it does not log anything.
+ 
+Also, it is possible to connect any logging library to Hazelcast Node.js client through adapters.
+
+See the following built-in `winston` logging library example.
```

Comment:
what does built-in mean here? It is not built-in to Hazelcast. It is an external library.

Reply:
I mean, it is a built-in Node.js library.

Reply:
It is not a built-in library. We add it explicitly as test dependency, just to test this feature. See: https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/package.json#L26